### ISSUE TITLE
Inquisition incorrectly increases holy damage 

### ISSUE THREAD

**Issue by Renehadox** [gamefreedomgit/Maelstrom#5512]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**

When a paladin uses inquisition and he stops to dps the target the target will not take increased damage until the paladin hits the target this is a major bug in arena when you are on 1 v 1 situations and for an example the target is a rogue and he is low hp, you use inquisition but you cant hit him and he wont take increased damage.

TL DR its a bug.

**How to reproduce:**

**How it should work:**

**Database links:**

**Comment by Sketish** [comment]
But from what he should take increased dmg ?  Censure stacks refresh on hti with increased dmg. And again start provifing proofs or evidence beacuse otherwise will mark those reports as invalid

**Comment by Renehadox** [comment]
> But from what he should take increased dmg ? Censure stacks refresh on hti with increased dmg. And again start provifing proofs or evidence beacuse otherwise will mark those reports as invalid

Censure stacks refresh on hti with increased dmg true but the mechanic is not supposed to work like that.
Once you use inquisition the target should start taking increased damage on the next censure tick and NOT ONCE YOU HIT the target again in order for it to work.

Its just that the mechanic is bugged.
Inquisition as a spell itself works properly giving the 30% increased damage.
And no i can't find a retail video cause its only a whitemane specific bug, i looked some videos but didnt find the moment with the inq. up and down.

**Comment by Sketish** [comment]
Unless you provide any proff wont count this as bug, Since censure is a dot which has a snapshot. So it wont be automatically update dmg till you refreshh it with melee hit.

**Comment by mipanka** [comment]
If im not mistake, snapshot was added in mop

**Comment by Sketish** [comment]
In WoD dots started to updated every second, snapshot is working at least from wotlk till end of mop

**Comment by UltraNix** [comment]
According to above info - no bug here.

**Comment by Renehadox** [comment]
> Unless you provide any proff wont count this as bug, Since censure is a dot which has a snapshot. So it wont be automatically update dmg till you refreshh it with melee hit.

How do you want me to provide a proof of that bug?
Its whitemane exclusive bug only like the other 1000 of bugs that i reported.
When a paladin have 5 stacks of seal of truth on an enemy target and he uses inuqisition the next tick should be dealt with 30% more increased damage aka inquisition damage.
It works like that on apollo for an example.
